### PR TITLE
Remove bootstrap deps

### PR DESCRIPTION
**Pull request by kaytcat**
- Remove all bootstrap classes in order to allow our applications the option to remove bootstrap too. I also updated kununu-icons with a brand new shiny Tuv svg.

### CODE REVIEW

File: packages/kununu-footer/index.scss [hunk 1]
```diff
@@ -190,13 +215,13 @@
...
+    text-align: right;
     max-width: 550px;
   }
-
-  @media (max-width: $screen-sm-max) {
-    width: 100%;
-  }
 }
 
 div.title {
```
Comment (clopes-ventureoak) [inline packages/kununu-footer/index.scss]: Do we still need this rule?

File: packages/kununu-footer/index.scss [hunk 2]
```diff
@@ -116,6 +118,27 @@
...
+  padding: 0;
+  letter-spacing: 0;
+  line-height: $line-height-h3;
+  font-weight: 600;
+  display: block;
+  font-size: 18px;
+  margin-bottom: $baseline-grid * 2;
+  margin-top: $baseline-grid * 3;
+
+  @media (max-width: $screen-sm-max) {
```
Comment (clopes-ventureoak) [inline packages/kununu-footer/index.scss]: Should we follow a "mobile first" approach?

File: packages/kununu-footer/index.scss (hunk 1 above)
Comment (kaytcat) [reply packages/kununu-footer/index.scss]: Hmm yes, but not here. I fixed it! Thanks for catching it 👍

File: packages/kununu-footer/index.scss (hunk 2 above)
Comment (kaytcat) [reply packages/kununu-footer/index.scss]: Yes, I suppose it could use a refactor to mobile first. I will update this.

**Pull request merged**